src/utils/DockerMachineUtil.js

```diff
@@ -60,31 +58,29 @@ var DockerMachine = {
       return false;
     });
   },
-  create: function (machineName = this.name(), provider) {
+  create: function (machineName = this.name(), provider = "hyperv", virtualBoxIsInstalled = true) {
```

Comment:
`virtualboxinstalled` is only needed as at that step you might not have it installed and we need to through an Exception and stop the whole process.